src/main/java/httpServer/routes/Router.java

```diff
@@ -54,8 +54,13 @@ private HTTPResponseBuilder setBody(HTTPRequest httpRequest, HTTPResponseBuilder
         if (httpRequest.resource.equals("/echo_body")) {
             responseBuilder = responseBuilder.setBody(httpRequest.body);
             return responseBuilder;
+        } else if (httpRequest.resource.equals("/json_response")) {
+            responseBuilder = responseBuilder.setBody("{\"key1\":\"value1\",\"key2\":\"value2\"}");
+            return responseBuilder;
+        } else {
```

Comment:
Might be time to change this to a `switch` statement: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html